api/services/gameService.js

```diff
@@ -117,9 +118,15 @@ module.exports = {
                   message: "Can't populate game, because it does not have players collection",
                 });
               })
+              // find spectators
+              .then(function findSpectators(values) {
+                const [game, p0, p1] = values;
+                const { spectators } = game;
+                return [game, p0, p1, spectators];
+              })
```

Comment:
We don't need a separate `.then()` handler for this because these are all synchronous operations to just read existing values
```suggestion
              const { spectators } = game;
```